lib/resources/elasticsearch.rb

```diff
@@ -0,0 +1,140 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Rony Xavier, rx294@nyu.edu
+# author: Aaron Lippold, lippold@gmail.com
+# license: All rights reserved
+
+require 'utils/filter'
+
+module Inspec::Resources
+  class Elasticsearch < Inspec.resource(1)
+    name 'elasticsearch'
+    desc 'See Description below'
```

Comment:
The `desc` contents are displayed in the `inspec shell` output... so there would be no description printed. Please uncomment similar to other resources.